doc/source/architecture.rst

```diff
@@ -35,6 +40,12 @@ The *indexer* is responsible for storing the index of all resources, archive
 policies and metrics, along with their definitions, types and properties. The
 indexer is also responsible for linking resources with metrics.
 
+The *coordinator* is responsible of the scheduling of jobs between *metricd*
+workers.
+
+The *notifier* is responsible of letting me the worker that they have metrics
+to compute.
```

Comment:
s/responsible of letting me the worker that they have metrics to compute/responsible for event-driven processing of incoming metrics/

i think we should be more explicit that this is an alternative to polling. for example, we should say if you enable notificatier, you should set polling to be less.